### PR TITLE
chore: prepare instruction for generating ML data

### PR DESCRIPTION
**Pull request by jeertmans**
Hello @nicoladicicco,

I added some documentation in current ML notebook about how input and output data are structured.

Note that there is a small caveat about how paths vertices are returned: put simply, you cannot just concatenate `[tx, path.vertices, rx]` altogether because one needs first to apply a mask (this is in addition to the padding values, that are used for something else).

I let you read the document, as well as this function that shows how to post-process paths (here, it is used to plot them):

https://github.com/jeertmans/sionna/blob/1f6bd43daaaad54c91702fb282aa36cd76d12b18/sionna/rt/utils.py#L270-L311

Let me know if you have any question!

### CODE REVIEW

**Review by nicoladicicco — APPROVED** [review]
I think I got it. Thank you very much for your clarification.

Regarding a couple of notes you left in the notebook:
> NOTE: the problem should be separable for each pair of (TX, RX). I.e., addressing the problem of path finding as if there were only one TX and one RX is ok.
This is what I had in mind for the neural net, i.e., to condition it to the TX-RX info. If one wants to evaluate multiple TX-RX locations this should be trivially parallelizable on GPU via batching.

> NOTE: it could be interesting to favor the paths that have the shortest length.
This is something that can be incorporated in the loss function.

**Comment by jeertmans** [comment]
Nice, thanks for reviewing it!
